operator/pkg/certs/certs.go

```diff
@@ -473,8 +473,8 @@ func apiServerAltNamesMutator(cfg *AltNamesMutatorConfig) (*certutil.AltNames, e
 			"kubernetes",
 			"kubernetes.default",
 			"kubernetes.default.svc",
-			fmt.Sprintf("*.%s.svc.cluster.local", cfg.Namespace),
-			fmt.Sprintf("*.%s.svc", cfg.Namespace),
+			fmt.Sprintf("*.%s.svc.cluster.local", constants.KarmadaSystemNamespace),
```

Comment:
Indeed, we need to add dns `*.karmada-system.svc.cluster.local` to certs.
Whether we should keep the previous dns？

Reply:
In fact, previous dns is not used. I will post the verification report later

Reply:
/cc @zhzhuang-zju Have you ever tested creating a pp?

Reply:
Not yet, have you found any problems?